### PR TITLE
Fb/addflower

### PR DESCRIPTION
**Pull request by NadyaSukhova**

### CODE REVIEW

File: aqi_monitor/apps/monitor/tasks.py [hunk 1]
```diff
@@ -9,23 +9,18 @@
 from . import parse_aqi_data
 
 
-app.conf.beat_schedule = \
-    {"send_message": {
-        "task": "apps.monitor.tasks.extract_aqi",
-        "schedule": crontab(minute="*/1"),
-        "args": (),
-    }}
-
```
Comment (il1tvinov) [inline aqi_monitor/apps/monitor/tasks.py:18]: @NadyaSukhova why did you remove that?

File: aqi_monitor/apps/monitor/parse_aqi_data.py [hunk 2]
```diff
@@ -62,10 +62,12 @@ def parse_aqi_data(data: dict) -> dict:
...
-              "aqi": data["aqi"],
-              "message": message_about_air_quality(data["aqi"]),
-              "time": time_convert_to_utc(data["time"])}
+    aqi = parse_iaqi(data.get("iaqi"))
+    result = {
+        "id_region": data.get("idx"),
+        "aqi": data.get("aqi"),
+        "message": message_about_air_quality(data.get("aqi")),
+        "time": time_convert_to_utc(data.get("time")),
+    }
```
Comment (il1tvinov) [inline aqi_monitor/apps/monitor/parse_aqi_data.py:71]: @NadyaSukhova do we really need this kind of approach? I think this particular function is unnecessary. Probably you can move parsing from `result` to `parse_iaqi` and remove this function and use `parse_iaqi` instead.  
What do you think?

File: aqi_monitor/apps/monitor/tasks.py (hunk 1 above)
Comment (NadyaSukhova) [reply aqi_monitor/apps/monitor/tasks.py]: accidentally, sorry
